### PR TITLE
FEAT: Support deepcache with sd models

### PR DESCRIPTION
**Pull request by frostyplanet**
If package "deepcache" is installed,turn on deepcache optimization by default, which brings 2x speedup.
The feature can be turned off by option 'deepcache=False'.

deepcache & cpu_offloading is tested.

### CODE REVIEW

File: xinference/model/image/stable_diffusion/core.py [hunk 1]
```diff
@@ -175,6 +175,17 @@ def load(self):
...
+        if self._kwargs.get("deepcache", True):
+            # NOTE: DeepCache should be loaded first before cpu_offloading
+            try:
+                from DeepCache import DeepCacheSDHelper
+
+                helper = DeepCacheSDHelper(pipe=self._model)
+                helper.set_params(cache_interval=3, cache_branch_id=0)
+                helper.enable()
+            except:
+                pass
```
Comment (qinxuye) [inline xinference/model/image/stable_diffusion/core.py:187]: Please add log to record the exception.

Comment (frostyplanet) [reply xinference/model/image/stable_diffusion/core.py:188]: fixed

File: xinference/model/image/stable_diffusion/core.py [hunk 2]
```diff
@@ -175,6 +175,18 @@ def load(self):
...
+            # NOTE: DeepCache should be loaded first before cpu_offloading
+            try:
+                from DeepCache import DeepCacheSDHelper
+
+                helper = DeepCacheSDHelper(pipe=self._model)
+                helper.set_params(cache_interval=3, cache_branch_id=0)
+                helper.enable()
+            except ImportError:
+                logger.debug("deepcache is not installed")
+                pass
```
Comment (qinxuye) [inline xinference/model/image/stable_diffusion/core.py:188]:
```suggestion
```

**Review by qinxuye — APPROVED** [review]
LGTM

**Review by qinxuye — APPROVED** [review]
LGTM

**Pull request merged**